src/app/items-list/items-list.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="itemsData">
```

Comment:
Maybe we could add an empty state handling here?
Please remember to add it to the task specs.